proj/cmake/modules/cinderMakeApp.cmake

```diff
@@ -61,12 +61,42 @@ function( ci_make_app )
 		set_source_files_properties( ${ARG_RESOURCES} PROPERTIES HEADER_FILE_ONLY ON MACOSX_PACKAGE_LOCATION Resources )
 	elseif( CINDER_LINUX )
 		unset( ARG_RESOURCES ) # Don't allow resources to be added to the executable on linux
+	elseif( CINDER_MSW )		
+		if( MSVC )
+			# Override the default /MD with /MT
+			foreach( 
+				flag_var
+				CMAKE_C_FLAGS CMAKE_C_FLAGS_DEBUG CMAKE_C_FLAGS_RELEASE CMAKE_C_FLAGS_MINSIZEREL CMAKE_C_FLAGS_RELWITHDEBINFO 
+				CMAKE_CXX_FLAGS CMAKE_CXX_FLAGS_DEBUG CMAKE_CXX_FLAGS_RELEASE CMAKE_CXX_FLAGS_MINSIZEREL CMAKE_CXX_FLAGS_RELWITHDEBINFO 
+			)
+				if( ${flag_var} MATCHES "/MD" )
+					string( REGEX REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}" )
+					set( "${flag_var}" "${${flag_var}}" PARENT_SCOPE )
+				endif()
+			endforeach()
+			# Force synchronous PDB writes
+			add_compile_options( /FS ) 
+			# Force multiprocess compilation
+			add_compile_options( /MP )
+			# Add necessary platform libraries		
+			list( APPEND PLATFORM_LIBRARIES "zlib.lib;shlwapi.lib;OpenGL32.lib" )
+			# Add lib dirs
+			cmake_policy( PUSH )
+			cmake_policy( SET CMP0015 OLD )
+			link_directories( "${CINDER_PATH}/${CINDER_LIB_DIRECTORY}" )
```

Comment:
There is not `target_*` variant for `link_directories` but what I am actually wondering is if this is needed at all. In theory ( and in practice as it happens for Linux / OS X for example ) the previous call to `find_package( cinder )` together with the `target_link_libraries( myapp cinder )` that follows should take care of finding and linking with Cinder without the need for manually specifying link directories.. But maybe that is not the case for the Visual Studio generator?